mongodb/devhub
# DEVHUB-892: Auto-Sign In

## Links:

-   [JIRA Ticket](https://jira.mongodb.org/browse/DEVHUB-892)
-   Please test locally as the local domains (and Kanopy staging) are only ones allowed by CORS

## Description:

-   This PR is for pulling in a user's session from Okta if they are already signed in to sign them in automatically. This is blocked until Cloud adds our domains (to resolve CORS issues when using their API)

## Testing

-   Logout by visiting /logout on DevHub
-   Verify you are not logged into DevHub
-   Login to https://account-qa.mongodb.com/account/login
-   See now you are signed into DevHub (user is populated in Authentication Context)

## What types of outside events need to happen for this PR?

-   [ ] Release Note Update (only for deployments)
